src/components/App/QRGenerator/index.tsx

```diff
@@ -0,0 +1,44 @@
+import * as React from 'react';
+import { useQRCode } from 'react-qrcodes';
+
+const QRCode = ({ text }: { text: string }) => {
+  const options = {
+    level: 'M',
+    margin: 4,
+    scale: 4,
+    width: 4,
+    color: {
+      dark: '#000000ff',
+      light: '#ffffffff'
+    }
+  };
+
+  const [qrRef] = useQRCode({
+    text: text,
```

Comment:
Should this be called text? Or would something like `link` be better? (applies to ever `text` variable in the file)

Reply:
Yeah because it can basically be anything, I think we need a discussion for whether we want it to be a link or not, then I'll add a regexp to filter links.